### DISCUSSION TITLE
feat(TreeView):希望给树形组件添加连接线

### DISCUSSION THREAD

**Discussion by Idle-wang** [dotnetcore/BootstrapBlazor#4416] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

希望给树形组件添加连接线，这样看起来结构会更清晰，如下图所示
![image](https://github.com/user-attachments/assets/671f112d-893d-436e-8421-8361b1512fbf)

**Comment by bb-auto** [comment]
@Idle-wang Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@Idle-wang 如果用户使用样式更改了行高或者字体大小，这些连接线如何处理？
